forge/test/mlir/test_ops.py

```diff
@@ -583,7 +583,45 @@ def forward(self, x, y):
 
     co_out = [co.to("cpu") for co in co_out]
     fw_out = [fw_out] if isinstance(fw_out, torch.Tensor) else fw_out
-    assert all([compare_with_golden_pcc(golden=fo, calculated=co, pcc=0.99) for fo, co in zip(fw_out, co_out)])
+    assert all([compare_with_golden(golden=fo, calculated=co) for fo, co in zip(fw_out, co_out)])
+
+
+@pytest.mark.parametrize(
+    "shape",
+    [
+        (1, 128, 28, 28),
```

Comment:
Can we test out different ranks as well? If some of them are not working, let's mark them as xfail.